<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>doctor_manage</class>
 <widget class="QMainWindow" name="doctor_manage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>981</width>
    <height>769</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Doctor Manage</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>10</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Doctors</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="table_doctor">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>831</width>
      <height>281</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>ID</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Name</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Phone</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Email</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Adress</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Hospital</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Edit</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Delete</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="btn_import_doctor">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>90</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Import</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_export_doctor">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>150</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Export</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tab_search_add_doctor">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>370</y>
      <width>941</width>
      <height>351</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab_search_doctor">
     <attribute name="title">
      <string>Search</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>811</width>
        <height>141</height>
       </rect>
      </property>
      <property name="title">
       <string>Search Doctor</string>
      </property>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>40</y>
         <width>55</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Name:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="txt_search_doctor">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>30</y>
         <width>661</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_search_doctor">
       <property name="geometry">
        <rect>
         <x>330</x>
         <y>90</y>
         <width>151</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Search Doctor</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_add_new_doctor">
     <attribute name="title">
      <string>Add new</string>
     </attribute>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>40</y>
        <width>55</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Name:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>90</y>
        <width>55</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Phone:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>140</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Email</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>190</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Address</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txt_name_doctor">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>31</y>
        <width>621</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="txt_phone_doctor">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>80</y>
        <width>621</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="txt_email_doctor">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>130</y>
        <width>621</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="txt_address_doctor">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>180</y>
        <width>621</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_add_doctor">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>280</y>
        <width>89</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Add</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>240</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Hospital id:</string>
      </property>
     </widget>
     <widget class="QComboBox" name="combo_hospital_doctor">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>231</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>981</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
